server/lobbyconnection.py

```diff
@@ -1031,6 +1031,13 @@ def write_launch_game(
         is_host: bool = False,
         options: GameLaunchOptions = GameLaunchOptions(),
     ) -> None:
+        if self.game_connection is not None:
+            self._logger.warning(
+                "%s launched a new game while old GameConnection was active",
```

Comment:
There are a few ways I think this could happen:
1. The game hard crashes, so it doesn't send `GameState: Ended` on exit. Maybe the ICE adapter will detect that and still send the command though, idk. Either way, I guess the ICE adapter could crash as well and then we'd have the same problem.
2. The server sends a `game_launch` command but for whatever reason the client doesn't actually open the game. Because the way the server code is written, a `GameConnection` object has to exist in order to receive GPGNet commands however, we don't actually know for sure if the game has opened until we receive a GPGNet command. This is the most likely scenario, in which case we can safely just remove the old `GameConnection` obejct.